eng/common/scripts/Invoke-GitHub-API.ps1

```diff
@@ -0,0 +1,180 @@
+param(
+  $AuthToken
+)
+
+$Token = ConvertTo-SecureString -String $AuthToken -AsPlainText -Force
```

Comment:
@mitchdenny do you think using secure string and passing separate parameters is easier or better then just passing the headers directly?

Reply:
@mitchdenny

Reply:
I'm not that hung up on it, but I think given that the Cmdlet has support for exactly what we want to do we should use it. I'm not a fan of secure string either since in our use case its not secure.